### DISCUSSION TITLE
AppImages & SquashFS

### DISCUSSION THREAD

**Discussion by WaveDevelops** [divestedcg/Brace#6] [Ideas] [answered]
Hello. :)

Are there concerns about SquashFS and/or AppImages that make them unusuable with Brace installed? The code references this page for disabling SquashFS:
https://madaidans-insecurities.github.io/guides/linux-hardening.html#kasr-kernel-modules

Tho it doesn't go into why, except to call it a "rare filesystem", tho with AppImages and Snaps gaining popularity I don't think that's really true anymore.

**Comment by SkewedZeppelin** [comment, marked as answer]
hm, it may indeed break those. but I don't necessarily recommend those either.

I'll comment it out for next update.

**Reply by WaveDevelops** [reply]
Yeah I'm not interested in Snaps myself, though I have noticed a handful of apps who's primary distribution method is AppImages, which is what prompted me to ask the question. My understanding was that they're pretty decent - and I believe Firejail supports them directly so I thought it might be worth reconsidering. This isn't particularly important to me though, so I'm not bothered either way if ya think they should be avoided.

In any case, thanks for the quick response! :)
